### PR TITLE
fixed the misplaced forgot password

### PR DESCRIPTION
**Pull request by shonali2600**
## Related Issuse
forgot password was with sign up

Closes: #100 

#### Describe the changes you've made
placed the forgot password to the correct place

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -1,312 +1,314 @@
...
+                                    <div>
+                                      {% for message in messages %}
+                                      {% if message.tags == 'error' %}
+                                      <div class="alert alert-danger" style="margin-top:10px">{{ message }}</div>
+                                      {% endif %}
+                                      {% endfor %}
+                                    </div>
+                                    {% endif %}
+                                </div>
+                                <i class="ion-ios-arrow-right"></i> <a href="{% url 'forgotp' %}">Forgot Password?</a>
```
Comment (khannakshat7) [inline templates/login.html:169]: Can u pl. remove this line from signup form

Comment (khannakshat7) [reply templates/login.html:169]: 169 or 170 pl check

Comment (shonali2600) [reply templates/login.html]: done sir

**Comment by shonali2600** [comment]
@khannakshat7 , Kindly merge my PR or let me know if any further changes to be made.

**Comment by khannakshat7** [comment]
@shonali2600 Pl. change only one the few lines that are required for this issue. I can see whole file changed in changes tab of this PR

**Comment by shonali2600** [comment]
@khannakshat7 earlier it was just 2 lines changes, but I think it's due to solving the conflicts

**Comment by shonali2600** [comment]
@khannakshat7, it's only 2 lines changes. If you check the code after clicking on `hide the whitespace changes` it shows the actual changes. 
electra

**Comment by khannakshat7** [comment]
@shonali2600 I think your fork is not updated. I suggest you fork it again and make changes. I'll merge it as quickly

**Comment by shonali2600** [comment]
@khannakshat7 I fetched the updated code and again made changes, now it's showing the correct changes

**Review by khannakshat7 — APPROVED** [review]

**Comment by khannakshat7** [comment]
@shonali2600 Awesome !! pl. contribute more like this🙌

**Comment by khannakshat7** [comment]
@shonali2600 Don't forget to star the repo⭐ and follow us on GitHub.

**Pull request merged**